.gitignore

```diff
@@ -1,4 +1,5 @@
 # build
+/build
```

Comment:
Build is the deafult name for the Ladle build of the stories

Reply:
ok, thank's

Reply:
By the way, try Building the stories. On my first attempt it failed.
Not so sure if building it's a priority since on dev you can do without it.
Anyways, that could be the best way of providing an interactive documentation on the library (like as static files, literally hosted anywhere)

Reply:
oh, I missed that comment, sorry. ok, I will check it